### PR TITLE
Release v0.23.11

### PR DESCRIPTION
**Pull request by rossabaker**
Need to land the plane sometime.  I expect another release train next week.

### CODE REVIEW

File: site/src/main/mdoc/changelog.md [hunk 1]
```diff
@@ -4,6 +4,60 @@
...
 
+# v0.23.11 (2022-03-18)
+
+This is a maintenance release, binary compatible with the 0.23.x series.  It also includes the changes in 0.22.12.
+
+* http4s-server
+    * Add Additional ErrorHandling Options by @ChristopherDavenport in https://github.com/http4s/http4s/pull/6047
+
+* http4s-client
+    * Make `WSClient` and friends public by @armanbilge in https://github.com/http4s/http4s/pull/6005
```
Comment (armanbilge) [inline site/src/main/mdoc/changelog.md:15]: Finally! 😆

**Review by armanbilge — APPROVED** [review]

**Review by danicheg — APPROVED** [review]

**Comment by armanbilge** [comment]
Hold, publishing is broken on series/0.23 darn it. We got bit by https://github.com/typelevel/sbt-typelevel/issues/214.

**Comment by rossabaker** [comment]
For my education, where do you see that it bit?  All I see is green.

**Comment by armanbilge** [comment]
The snapshot publish failed on series/0.23.
https://github.com/http4s/http4s/runs/5601683107?check_suite_focus=true

**Comment by armanbilge** [comment]
> All I see is green.

Funny enough, that is precisely what the bug in https://github.com/typelevel/sbt-typelevel/issues/214 is about 😆

**Comment by rossabaker** [comment]
This is what I get for commenting while in a meeting.

**Review by ChristopherDavenport — APPROVED** [review]

**Comment by armanbilge** [comment]
`0.23.10-215-351838d-SNAPSHOT` published successfully. So I'd say we're good on 0.23.11.

**Pull request merged**